### PR TITLE
Stories mod

### PR DESCRIPTION
**Pull request by Velenir**
- Extend `@storybook` types to have `Template.bind({}).args` already typed
- Global decorators with `<GlobalStyles/>`
- markdown rules + `FAQ.story`

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1354--dexreact.review.gnosisdev.com

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+import React from 'react'
+import GlobalStyles from 'styles/global'
+
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-}
\ No newline at end of file
+}
+
+export const decorators = [
```
Comment (anxolin) [inline .storybook/preview.js:9]: nice! 

So this file still cannot easily be a TS? (I mean without nast workarounds)

File: src/types/storybook.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { Story } from '@storybook/react/types-6-0'
+declare module '@storybook/addons' {
+  export interface BaseStory<Args, StoryFnReturnType> {
+    bind(thisArg: Parameters<Function['bind']>[0]): Story<Args>
```
Comment (anxolin) [inline src/types/storybook.d.ts:4]: I'm lost in this change. What is this for? 
Maybe, add it also as a comment to

**Review by anxolin — APPROVED** [review]
Nice one!

File: src/types/storybook.d.ts (hunk 2 above)
Comment (Velenir) [reply src/types/storybook.d.ts:4]: so you don't have to do `.args = {} as Props`

File: .storybook/preview.js (hunk 1 above)
Comment (Velenir) [reply .storybook/preview.js]: I just tried and turns out you can a23efe9

File: src/types/storybook.d.ts (hunk 2 above)
Comment (alfetopito) [reply src/types/storybook.d.ts:4]: :open_mouth:

Comment (anxolin) [reply src/types/storybook.d.ts:4]: 对我来说那是中国人，迪玛

**Pull request merged**